client/src/main/scala/drt/client/components/MultiDayExportComponent.scala

```diff
@@ -74,23 +79,18 @@ object MultiDayExportComponent extends WithScalaCssImplicits {
 
       val setStartDate: ReactEventFromInput => CallbackTo[Unit] = e => {
         e.persist()
-        isNotValidDate(e.target.value) match {
-          case true =>
-            scope.modState(_.setStart(e.target.value, true))
-
-          case _ =>
-            scope.modState(_.setStart(e.target.value, false))
-        }
+        if (isNotValidDate(e.target.value))
+          scope.modState(_.setStart(e.target.value, isNotValid = true))
+        else
+          scope.modState(_.setStart(e.target.value, isNotValid = false))
```

Comment:
This could be improved - we're checking if the date is valid, and if it isn't then we're sending to as the new date anyway, which then gets ignored in the `setStart` function